PubMed XML.js

```diff
@@ -0,0 +1,566 @@
+{
+	"translatorID": "fcf41bed-0cbc-3704-85c7-8062a0068a7a",
+	"label": "PubMed XML",
+	"creator": "Simon Kornblith, Michael Berkowitz, Avram Lyon, and Rintze Zelle",
+	"target": "xml",
+	"minVersion": "2.1.9",
+	"maxVersion": "",
+	"priority": 100,
+	"configOptions": {
+		"dataMode": "xml/dom"
+	},
+	"inRepository": true,
+	"translatorType": 1,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-03-21 12:27:58"
+}
+
+/*******************************
+ * Import translator functions *
+ *******************************/
+
+function detectImport() {
+	Zotero.debug("Detecting Pubmed content....");
+	// Look for the PubmedArticle tag in the first 1000 characters
+	var text = Zotero.read(1000);
+	if (text.indexOf("<PubmedArticle>") != -1) return "journalArticle";
+	if (text.indexOf("<PubmedBookArticle>") != -1) return "book";
```

Comment:
For import you only need to return `true` or `false`. So just look for `<PubmedArticleSet>`